dbfit-java/informix/src/test/java/dbfit/environment/InformixBigDecimalTransformerTest.java

```diff
@@ -0,0 +1,42 @@
+package dbfit.environment;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.lang.Double;
+import java.sql.SQLException;
+
+import dbfit.util.NormalisedBigDecimal;
+
+public class InformixBigDecimalTransformerTest {
+
+    private final InformixBigDecimalTransformer bdt = new InformixBigDecimalTransformer();
+    private final BigDecimal inBigDec = new BigDecimal("123.45");
+
+    @Test
+    public void transformsBigDecimaltoBigDecimalTest() throws SQLException {
+        BigDecimal bd = new BigDecimal(inBigDec.toString());
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(bd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void transformsNormalisedBigDecimaltoBigDecimalTest() throws SQLException {
+        NormalisedBigDecimal nbd = new NormalisedBigDecimal(inBigDec);
+        BigDecimal outBigDec = (BigDecimal) bdt.transform(nbd);
+        assertEquals(outBigDec, inBigDec);
+    }
+
+    @Test
+    public void rejectsNonNormalisedBigDecimalInputTest() {
```

Comment:
> I'm not quite sure how this `@Rule` annotation works. Does it set the expected exception (to no exception) for all of the test cases in the test class, or just those that follow it in the order of the code?

Ok, so `public ExpectedException exception = ExpectedException.none();` is initialising the `exception` then expected behaviours get redefined in the actual test methods.